flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/SubtaskStateStats.java

```diff
@@ -75,7 +75,6 @@
 
         checkArgument(subtaskIndex >= 0, "Negative subtask index");
         this.subtaskIndex = subtaskIndex;
-        checkArgument(stateSize >= 0, "Negative state size");
```

Comment:
Why was this removed?

Reply:
I removed it for debugging, will restore. Thanks for pointing out!